.github/workflows/appsignals-e2e-test.yml

```diff
@@ -62,18 +51,13 @@ jobs:
             scripts/eks/appsignals/clean-app-signals.sh
           sparse-checkout-cone-mode: false
 
-      - uses: actions/setup-java@v4
-        with:
-          java-version: 17
-          distribution: temurin
-
       - name: Generate testing id
         run: echo TESTING_ID="${{ env.AWS_DEFAULT_REGION }}-${{ github.run_id }}-${{ github.run_number }}" >> $GITHUB_ENV
 
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v4
         with:
-          role-to-assume: ${{ secrets.APP_SIGNALS_E2E_TEST_ROLE_ARN }}
+          role-to-assume: arn:aws:iam::${{ env.TEST_ACCOUNT }}:role/${{ secrets.APP_SIGNALS_E2E_TEST_ROLE_ARN }}
```

Comment:
`APP_SIGNALS_E2E_TEST_ROLE_ARN` is no longer accurate if we code the first half of the ARN. The secret name should now be `APP_SIGNALS_E2E_TEST_ROLE_NAME` or something of the sort.

Reply:
I'll ask Lisa to update it tmr

Reply:
Changed